ComSemApp/student/views.py

```diff
@@ -798,7 +798,7 @@ def get_context_data(self, **kwargs) -> dict[str, dict[str, str | int | float]]:
         """
         context : dict[str, Any] = super(SpeakingPracticeResultsView, self).get_context_data(**kwargs)
         context['attempts'] : list[dict[str, str | float]] = {}
-        attempt_ids : list[str] = dict(self.request.GET)['attempt']
+        attempt_ids : list[str] = dict(self.request.GET)['attempt_ids']
```

Comment:
This is a good catch, however: if you look at the url, it looks a bit messy with several `attempt_ids` in the GET request. It may be better to change it to just `id` rather than attempt so it's more concise.
Obviously, this would require a small frontend refactor.

Reply:
gonna make this a new issue aswell. I just want to get these changes merged in already so I have a consistent working branch for testing.

Reply:
created issue #209 to deal with this later